src/eherkenning/templates/eherkenning/mock/eherkenning_base.html

```diff
@@ -0,0 +1,79 @@
+{% load static %}
+<!DOCTYPE html>
+<html class='no-javascript' lang='nl'>
+<head>
+    <meta charset='utf-8'/>
+    <meta
+        content='Met eHerkenning kunt u zich identificeren op websites van de overheid. Zo weten overheidsinstellingen dat ze echt met u te maken hebben.'
+        name='description'/>
+    <meta content='eHerkenning, inloggen, aanvragen, activeren, Mijn DigiD' name='keywords'/>
+    <meta content='index, follow' name='robots'/>
+    <meta content='width=device-width, initial-scale=1' name='viewport'/>
+    <meta content='none' name='msapplication-config'/>
+
+    <title>{{ page_title }}</title>
+
+    <link href='{% static 'digid-mock/assets/ro-favicon-wit-0xffffff.png' %}' rel='shortcut icon'/>
+    <link rel="stylesheet" media="all" href="{% static 'digid-mock/assets/application.css' %}"/>
+    <link rel="stylesheet" media="all" href="{% static 'digid-mock/extra.css' %}"/>
+</head>
+<body>
+<div class='small window'>
+    <div class='content'>
+        <header class='header'>
+            <div>
+                {% block service_menu %}{% endblock %}
+                <a class="digidmock-anchor-placeholder">
+                    <img alt="Logo Rijksoverheid" class="rijksoverheid-logo" title="Logo Rijksoverheid"
+                        src="{% static 'digid-mock/assets/RO_DigiD_Logo_Homepage.svg' %}"/>
+                </a>
+                <nav class='header-navigation'></nav>
+            </div>
+            <div style='margin-top: 15px'></div>
```

Comment:
Styling for this and the next template is partly or completely (?) inline. Is this preliminary or the intended result for these templates?

Reply:
I copied the template from the DigiD mock flow from the `digid-eherkenning` library, to not spend too much time on the looks of it, since I think this template in particular probably needs attention later on if we want to get this into the library

Reply:
Styling this form is out-of-scope as there isn't a single eHerkenning look-and-feel. A plain html form was the minimum, and this looks a lot neater.